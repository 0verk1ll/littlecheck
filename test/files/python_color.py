# RUN: /usr/bin/python %s
import sys
sys.stdout.write("\x1B[33mFOO\n")
# CHECK: banana
