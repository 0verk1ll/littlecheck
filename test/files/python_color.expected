Failure in python_color.py:

  The CHECK on line 4 wants:
    banana

  which failed to match line stdout:1:
    \x1b[33mFOO

  when running command:
    /usr/bin/python python_color.py
